### PR TITLE
FD 1.13.0 Project History - Draft 2

### PR DESCRIPTION
**Pull request by mcherem**
Edits to the doc against the SW.

### CODE REVIEW

File: modules/ROOT/pages/fd-project-history.adoc [hunk 1]
```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
...
-. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
-. Click a saved state from the list.
-. Inspect the saved state in the sidebar by clicking on an object in the flow. Use this method to inspect all of the objects in the flow.
-. Continue either selecting different saved states from the list or the objects within them until you locate the saved state you want to restore.
-. Click *Restore* at the top of the canvas.
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore to a previous version:
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: To restore a previous version of the project? "Restore" is a transitive verb and requires an object.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 2]
```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
...
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
+. Click a saved state from the list. The saved state is a read-only view of each card in the project flow on the canvas.
+. Click on a card to view the configuration details of that object in the flow. Continue with this method to review any changes for a card or object in the flow.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Perhaps line 24 could be "Click on any card to view the details of its configuration."?
About "Continue with this method": what method?
About "review any changes": are changes marked, so that users can see what changed?
About "for a card or object": What other objects besides cards? The first sentence in the step mentions only cards.
Both sentences end with "in the flow". Either use it in only one sentence or (better) drop it completely..

File: modules/ROOT/pages/fd-project-history.adoc [hunk 3]
```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
...
+. Once you decide upon and select a saved state and want to revert, click *Restore* at the top of the canvas. From the dialog that displays you have a choice to restore or cancel. Cancel returns you to the selected state.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: "Once you decide upon and select a saved state and want to revert": there is one too many use of "and" in that clause. Do you really need "decide upon and"?
Perhaps "After you select a state to revert your project to, click ..."?
About "From the dialog that displays you have a choice to restore or cancel.": The procedure is about how to restore a previous state of the project. So, don't mention the option to cancel. Almost every dialog has a Cancel button, so users will be well aware of what would happen if they were to click Cancel.I would strike out this sentence.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 4]
```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
...
+. Click *Restore* to restore to the state you selected and return to the Project Canvas. The restored state is moved to the top of the history list.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Should be "to restore the selected state of the project". 
What is the "Project Canvas"? If it's the canvas, you've referred to it elsewhere in this topic simply as the "canvas".

File: modules/ROOT/pages/fd-project-history.adoc [hunk 5]
```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
 
 Being able to view a saved state of a project is useful if you want to:
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Why does this sentence say "to _view_ a saved state"? The topic is about restoring a saved state of a project.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Shoot, this is still there. With all the back and forth comments last night, I must have missed this. I was trying not to use 'version'. Will use your suggestion.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Yes on 24. For this:
About "Continue with this method": what method? - what I was trying to say is that the user can continue to view the values of each card to see what's going on.  They aren't marked and there is no indication of any change. I have added some wording that perhaps fits. 
For this: About "for a card or object": What other objects besides cards? The first sentence in the step mentions only cards. - I'm still trying to solidify the terminology in my head. A card in the flow represents - a core component, an operation of a connector, a module, or an API. (Or so says the doc). Would object be a good overall term or do you suggest 
I specifically list these items? Are you saying that just using 'card' is not adequate here?
Removed flow.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Okay, will use: After you select a state to revert your project to
Okay, removing the cancel stuff. The user needs to click Restore twice (in two different places) to revert to the saved state. I thought it less confusing to explain the modal dialog.

Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: I just mention the confirmation dialog to avoid confusion. If you don't think that sentence should be there, I will remove it.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: AGain, me fumbling over terminology as I learn it. I have changed it to 'canvas' here.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Great catch, not sure why I had it that way in the first place.

Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Also, any thoughts on whether or not this is accurate?  'Retrieve previous values of connector configurations'. If not, what am I missing here?

File: modules/ROOT/pages/fd-project-history.adoc [hunk 6]
```diff
@@ -9,24 +9,18 @@ endif::[]
...
-. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
-. Click a saved state from the list.
-. Inspect the saved state in the sidebar by clicking on an object in the flow. Use this method to inspect all of the objects in the flow.
-. Continue either selecting different saved states from the list or the objects within them until you locate the saved state you want to restore.
-. Click *Restore* at the top of the canvas.
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore to a previous version of the project:
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: "Restore" is a transitive verb and requires a direct object. A prepositional phrase is not a direct object.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 7]
```diff
@@ -9,24 +9,18 @@ endif::[]
 You can see a history of changes you make to a project in Flow Designer.
 The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Sorry, but isn't this obvious: "The activity name is based on the action you performed at save time."? Or do you mean the name of the saved state?

File: modules/ROOT/pages/fd-project-history.adoc [hunk 8]
```diff
@@ -9,24 +9,18 @@ endif::[]
...
 
-Being able to view a saved state of a project is useful if you want to:
-
-* Undo an inadvertent change you made to a project by reverting back to a working state.
-* Review changes made by other users to the project.
-* Retrieve older values of some project configurations.
+Being able to restore a saved state of a project is useful if you want to:
 
+* Undo an inadvertent change you made to a project and revert to the working state.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Why did you change "a working state" back to "the working state"? There might be a number of saved states that were working.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 9]
```diff
@@ -9,24 +9,18 @@ endif::[]
...
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore to a previous version of the project:
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
+. Click a saved state from the list. The saved state is a read-only view of each card in the project on the canvas.
+. Click on any card to see the details of its configuration. This can be helpful to review the values. 
+. After you select a state to revert your project to, click *Restore* at the top of the canvas. You see the conformation dialog.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: "You see the conformation dialog." sounds awkward to me. I'm used to stating such things like this: _The confirmation dialog appears._

File: modules/ROOT/pages/fd-project-history.adoc [hunk 10]
```diff
@@ -9,24 +9,18 @@ endif::[]
...
+. Click *Restore* to restore to the state you selected and return to the canvas. The restored state is moved to the top of the history list.
```
Comment (rddheath) [inline modules/ROOT/pages/fd-project-history.adoc]: Again, "restore" takes a direct object. A prepositional phrase is not an object. I suggest "to restore the state that you selected..." Otherwise, the reader is left asking, "To restore what?"

File: modules/ROOT/pages/fd-project-history.adoc (hunk 9 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Yeah, think it's just a difference of writing styles. I was taught not to use: 'appears' because it does not translate well into Asian languages.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Duly noted about your concerns. I want to be sure to address your comments especially if they alter the technical meaning. But, if they are around language and style, then I can work with Kim on them.

Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: I have reworded it.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 7 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: I do not think it is obvious. Activity is one type of a saved state. I thought it helpful to clarify what activity meant in this context. But if you feel that part is easy for the user to understand, then I defer to you.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 8 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Not sure why this was changed back (as you say) or missed.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 9 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Changed to your suggestion.

File: modules/ROOT/pages/fd-project-history.adoc (hunk 10 above)
Comment (mcherem) [reply modules/ROOT/pages/fd-project-history.adoc]: Okay, I see what's going on here. You would like to see the action of Restore within the sentence, right?
To restore to the state that you selected, click Restore.
I think I'm coming from a fundamentally different place in my writing. We should discuss it sometime.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 11]
```diff
@@ -7,26 +7,22 @@ endif::[]
 :imagesdir: ../assets/images
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]: HIDDEN TEXT above graphic
No edits

= Discover What’s Changed in a Project
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords:

File: modules/ROOT/pages/fd-project-history.adoc [hunk 12]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
 
 You can see a history of changes you make to a project in Flow Designer.
-The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
+The saved states are listed by the ID of the user who made the change, time, date, and activity.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
The saved states are listed by the ID of the user who made the change, by time, by date, and by activity. 
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 13]
```diff
@@ -7,26 +7,22 @@ endif::[]
 :imagesdir: ../assets/images
 
 You can see a history of changes you make to a project in Flow Designer.
-The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
+The saved states are listed by the ID of the user who made the change, time, date, and activity. 
 
-Being able to view a saved state of a project is useful if you want to:
+The name of the saved state is based on the activity you performed at save time.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
The name of the saved state is based on the activity you performed when you saved the project.
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 14]
```diff
@@ -7,26 +7,22 @@ endif::[]
 :imagesdir: ../assets/images
 
 You can see a history of changes you make to a project in Flow Designer.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
You can see a history of changes you make to a project in Flow Designer. This enables you to revert to a previous version of your project, should you need to.
```

Comment (kbostrom) [reply modules/ROOT/pages/fd-project-history.adoc]: I just added the second part of that sentence to provide a link to the sudden comment about "restore a saved state" that comes later. I guessed. Adjust as necessary to retain intended tech meaning.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 15]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
-The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
+The saved states are listed by the ID of the user who made the change, time, date, and activity. 
 
-Being able to view a saved state of a project is useful if you want to:
+The name of the saved state is based on the activity you performed at save time.
 
-* Undo an inadvertent change you made to a project by reverting back to a working state.
-* Review changes made by other users to the project.
-* Retrieve older values of some project configurations.
+Being able to restore a saved state of a project is useful if you want to:
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]: Remove periods from the ends of these list items.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 16]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
 
+* Undo an inadvertent change you made to a project and revert to a working state.
+* Review changes made by other users on the project.
+* Retrieve previous values of connector configurations.
 
 == Procedure
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]: I suggest omitting this solitary subtitle.

File: modules/ROOT/pages/fd-project-history.adoc [hunk 17]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
-. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
-. Click a saved state from the list.
-. Inspect the saved state in the sidebar by clicking on an object in the flow. Use this method to inspect all of the objects in the flow.
-. Continue either selecting different saved states from the list or the objects within them until you locate the saved state you want to restore.
-. Click *Restore* at the top of the canvas.
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore your saved state to a previous version of the project:
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
To revert to a previous state of your project:
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 18]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
. Click *History* in the upper-left of the canvas to open the Project History list on the right.
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 19]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
+. Click a saved state from the list. The saved state is a read-only view of each card in the project on the canvas.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
. Click a saved state from the list. 
The saved state is a read-only view of each card in the project on the canvas.
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 20]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
+. Click on any card to see the details of its configuration. This can be helpful to review the values.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
. Click on any card to see the details of its configuration. 
This can be helpful to review the values. 
```
The values of what?

File: modules/ROOT/pages/fd-project-history.adoc [hunk 21]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore your saved state to a previous version of the project:
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
+. Click a saved state from the list. The saved state is a read-only view of each card in the project on the canvas.
+. Click on any card to see the details of its configuration. This can be helpful to review the values. 
+. After you select a state to revert your project to, click *Restore* at the top of the canvas. The conformation dialog appears.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]:
```suggestion
. Click *Restore* at the top of the canvas. 
A confirmation dialog appears.
```

File: modules/ROOT/pages/fd-project-history.adoc [hunk 22]
```diff
@@ -7,26 +7,22 @@ endif::[]
...
+. To restore to the state you selected and return to the canvas click *Restore*. The restored state is moved to the top of the history list.
```
Comment (kbostrom) [inline modules/ROOT/pages/fd-project-history.adoc]: The user has to click Restore twice? Review this step and the one above it for clarity.
```suggestion
. To restore to the state you selected and return to the canvas click *Restore*. 
The restored state is moved to the top of the history list.
```

File: modules/ROOT/pages/fd-project-history.adoc (hunk 14 above)
Comment (ryanhol) [reply modules/ROOT/pages/fd-project-history.adoc]: In my view, I think the fact that users can revert is clear from the rest of the doc.
